src/isyntax/isyntax.c

```diff
@@ -47,7 +47,7 @@
 
 */
 
-
```

Comment:
There already is ASSERT() from common.h, should we use that instead or should we replace all ASSERT() with assert()?

Reply:
Fixed to use common.h ASSERT().

(I have slight preference to standard solutions, but it is best being consistent with the existing codebase. This file uses ASSERT(), so I'll do the same).